lib/node/rpc.js

```diff
@@ -2166,6 +2169,11 @@ class RPC extends RPCBase {
     if (!ns || !ns.owner)
       throw new RPCError(errs.MISC_ERROR, 'Cannot find the name owner.');
 
+    const state = ns.state(height, network);
+
+    if (state < states.CLOSED)
```

Comment:
Technically `REVOKED` is > `CLOSED`, any reason why this isn't `===` ?

Reply:
You're. It should throw error if `!== CLOSED`. Added an extra test and fixed.